roles/matrix-bot-matrix-registration-bot/templates/env.j2

```diff
@@ -0,0 +1,6 @@
+BOT_SERVER={{ matrix_bot_matrix_registration_bot_bot_server }}
+BOT_USERNAME={{ matrix_bot_matrix_registration_bot_matrix_user_id_localpart }}
+BOT_PASSWORD={{ matrix_bot_matrix_registration_bot_matrix_user_password }}
+API_BASE_URL={{ matrix_bot_matrix_registration_bot_api_base_url }}
+API_TOKEN={{ matrix_bot_matrix_registration_bot_matrix_admin_token }}
+LOGGING_LEVEL={{ matrix_bot_matrix_registration_bot_logging_level }}
```

Comment:
Let's add an "additional variables" variable, like `matrix_bot_honoroit_environment_variables_extension`, so that users can put more variables here easily.

Reply:
Will do